src/python/bot/fuzzers/init.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzing engine initialization."""
+
+from bot.fuzzers import engine
+from bot.fuzzers.libFuzzer import engine as libFuzzer_engine
+
+
+def run():
+  """Initialise builtin fuzzing engines."""
+  engine.register('libFuzzer_test', libFuzzer_engine.LibFuzzerEngine)
```

Comment:
I am guessing this is temporary as you are testing things ? Will this start creating any testcase with libFuzzer_test engine ?

Reply:
Yes this is a temporary thing. Will only be enabled with a small number of test jobs marked as experimental.